app.py

```diff
@@ -32,7 +32,7 @@
 
 testing = "TESTING" in os.environ
 db_url = Configuration.database_url(testing)
-SessionManager.initialize(db_url, testing=testing)
+SessionManager.initialize(db_url)
```

Comment:
The testing flag is no longer necessary because SessionManager.initialize not longer needs it.